akka-http-core/src/main/scala/akka/http/javadsl/Http.scala

```diff
@@ -33,7 +33,7 @@ import akka.stream.scaladsl.Keep
 object Http extends ExtensionId[Http] with ExtensionIdProvider {
   override def get(system: ActorSystem): Http = super.get(system)
   override def get(system: ClassicActorSystemProvider): Http = super.get(system)
-  def lookup() = Http
+  def lookup = Http
```

Comment:
Technically this method is public, so maybe Mima will complain. Or maybe it shouldn't because it's an implementation and the final shape has already been determined by the trait it implements even in Scala 2?

Reply:
Mima doesn't complain, so :+1: